android/library/src/main/java/com/liferay/mobile/screens/ddm/form/DDMFormListener.kt

```diff
@@ -24,6 +24,6 @@ interface DDMFormListener {
 
 	fun onFormLoaded(formInstance: FormInstance)
 
-	fun onError(exception: Exception)
+	fun onError(exception: Throwable)
```

Comment:
why change exception to throwable?

Reply:
Yeah, now kotlin has a Result model, so we don't need a library to do that, but it uses Throwable instead of Exception by default.